client/src/containers/about/AuthorsFragment.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Row, Col, Typography } from 'antd';
+import { Trans } from '@lingui/macro';
+
+import MAME1 from '../../images/MAME1.png';
+import CfR from '../../images/CfR.svg';
+
+const { Paragraph } = Typography;
+
+export default () => {
+  return (
+    <Row type="flex" justify="space-around" style={{ margin: '1rem', textAlign: 'left' }}>
+      <Col sm={20} md={12} className="about-authors-col">
+        <Paragraph style={{ padding: '1.5rem' }}>
+          <Trans>Proiect realizat de</Trans>
+        </Paragraph>
+        <img src={MAME1} alt="Make it better" width="186.36px" height="80px" />
+      </Col>
+      <Col sm={20} md={12} className="about-authors-col">
+        <Paragraph style={{ padding: '1.5rem' }}>
+          <Trans>Proiectat de</Trans>
+        </Paragraph>
+        <img src={CfR} alt="Code for Romania" width="186.36px" height="80px" />
```

Comment:
You don't need to specify px unit in width and height. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width

Reply:
Also, the content is not aligned properly to the left of the page. Tested in Firefox 
![image](https://user-images.githubusercontent.com/282668/109799834-261cfc00-7c25-11eb-9f11-d8cdb454e693.png)

Reply:
In Safari and Chrome is aligned perfectly. I will install Firefox to see what is the problem.

Reply:
<img width="1280" alt="Screen Shot 2021-03-03 at 8 07 19 PM" src="https://user-images.githubusercontent.com/25109595/109851226-1c62bb00-7c5c-11eb-9eee-ccd62115a18b.png">

Firefox on MacOs Catalina. I can't fix a problem that I can't see. Can you please double check?

Reply:
Maybe is from my Linux machine. Alright then, we can merge this and if the problem persist on my side, I will open a new issue.